pom.xml

```diff
@@ -113,6 +113,11 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>mailer</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>jaxb</artifactId>
+            <version>2.3.6-1</version>
```

Comment:
Note: We will be able to (and should) remove this version line when the next release of bom is available: https://github.com/jenkinsci/bom/commit/c2934433526443288d7372658aa8a9fd72c76a05

Reply:
=> #73 CC @ikedam @DuMaM

Reply:
@basil @ikedam 
This plugin doesn't support java 8 anymore.
Can I remove this freely?

Reply:
No, it needs to stay because Java 11 and Java 17 don't bundle JAXB anymore (but this plugin needs JAXB via its dependency on `com.atlassian.crowd:crowd-integration-client-rest`).

Reply:
com.atlassian.crowd:crowd-integration-client-rest
As far as i know 5.1.0 version don't need it. :)

Reply:
Awww... you right. 👍
thanks ;)